metricbeat/mb/helper/url.go

```diff
@@ -0,0 +1,91 @@
+package helper
```

Comment:
I left the package inside of `mb` because it contains functions that apply specifically to the things in the `mb`. I did rename it from the generic `helpers` to `parse` which is more specific and less likely to collide with a future helpers package that may exist someday.